src/lib.rs

```diff
@@ -971,6 +980,21 @@ impl TestKit {
         );
         self.cfg_proposal = Some(Uncommitted(proposal));
     }
+
+    /// Returns the node in the emulated network, from whose perspective the testkit operates.
+    pub fn us(&self) -> &TestNode {
+        self.network().us()
+    }
+
+    /// Returns public key of the validator.
+    pub fn service_public_key(&self) -> crypto::PublicKey {
+        *self.network().validators()[0].service_keypair().0
```

Comment:
Frankly speaking, I find `service_public_key` and `service_secret_key` in `TestKit` strange and possibly misleading. I think `testkit.us().some_key()` or `testkit.validator(ValidatorId(id)).some_key()` could be more understandable and are only a bit more verbose. In particular, why is the first validator used here? This doesn't seem like an obvious behavior.